### PR TITLE
(DOCSP-12745) Fixing errors

### PR DESCRIPTION
**Pull request by atsansone**
@JuliaMongo , @melissamahoney-mongodb : Please review to see if these changes align with what was noted in [DOCSP-12745](https://jira.mongodb.org/browse/DOCSP-12745). Thanks!

These include:

- [Item 1](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-12745/style-guide/terminology/terms-for-global-audience/)

- [Item 2](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-12745/style-guide/style/capitalization/)

- [Item 3](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-12745/style-guide/writing/write-clear-concise-sentences-paragraphs/#avoid-expletive-structures)

- [Item 4](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-12745/style-guide/style/product-names-version-numbers.html)

- [Item 5](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-12745/style-guide/terminology/alphabetical-terms.html#term-login)

- [Item 6](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-12745/style-guide/writing/use-that-which-correctly/)

@melissamahoney-mongodb , Based on @JuliaMongo 's suggestion, I added [this section](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-12745/style-guide/terminology/terms-for-global-audience.html#use-non-oppressive-language). That's what I hope you could review.

---
[Builds without errors on `master`.](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5f9721f98f1fd0fda5d5ed28)

### CODE REVIEW

**Review by JuliaMongo — APPROVED** [review]
LGTM

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 1]
```diff
@@ -305,6 +301,68 @@ If you create "named" users for extended examples or scenarios, use
...
+.. _oppressive-language:
+
+Use Non-Oppressive Language
+---------------------------
+
+Language choice deal in connotation. To ignore that ignores the power
+of speech and writing. Consistent with our goal of empowering everyone
+to use our technology, we must consider our use of terms that have been
+identified as racist, sexist, or otherwise oppressive. Some terms lack
+any ambiguity as to whether or not they are racist.
```
Comment (melissamahoney-mongodb) [inline source/style-guide/terminology/terms-for-global-audience.txt:313]: This feels a little unnecessarily obfuscated. How about:
```suggestion
Consistent with our goal of empowering everyone
to use our technology, we must consider the 
connotation of the words we use and avoid terms 
that have been identified as racist, sexist, or 
otherwise oppressive. Some terms lack any ambiguity 
as to whether or not they are racist.
```

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 2]
```diff
@@ -305,6 +301,68 @@ If you create "named" users for extended examples or scenarios, use
...
+
+The following table lists the terms to be avoided and their acceptable
```
Comment (melissamahoney-mongodb) [inline source/style-guide/terminology/terms-for-global-audience.txt:315]:
```suggestion
The following table lists terms to avoid and some acceptable
```

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 3]
```diff
@@ -305,6 +301,68 @@ If you create "named" users for extended examples or scenarios, use
...
+
+   * - slave
+     - n, adj
+     - subordinate, secondary, standby, replica, worker
+     - Replica sets have one primary and two or more secondary nodes.
+
+   * - slave
+     - v
+     - subordinate
+     - Replica sets have one primary and two or more secondary nodes.
```
Comment (melissamahoney-mongodb) [inline source/style-guide/terminology/terms-for-global-audience.txt:351]: Should this be an example with verb usage?

**Review by melissamahoney-mongodb — COMMENTED** [review]
A few thoughts, but I think this is an excellent section to add!

File: source/style-guide/terminology/terms-for-global-audience.txt (hunk 1 above)
Comment (atsansone) [reply source/style-guide/terminology/terms-for-global-audience.txt:313]: Still a bit wordy. Substituting:
```
Language choice deals in connotation. To ignore that ignores the power of speech and writing. MongoDB wants to empower everyone to use our technology. Keeping with that goal, we must consider our use of terms identified as racist, sexist, or otherwise oppressive. Some terms lack any ambiguity; they're racist. 
```

Comment (melissamahoney-mongodb) [reply source/style-guide/terminology/terms-for-global-audience.txt:313]: I don't see this change?

**Review by jdestefano-mongo — CHANGES_REQUESTED** [review]
@atsansone Some suggestions, overall looking pretty good. Solid addition to the SG.

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 4]
```diff
@@ -305,6 +301,68 @@ If you create "named" users for extended examples or scenarios, use
...
+
+   * - slave
+     - n, adj
+     - subordinate, secondary, standby, replica, worker
+     - Replica sets have one primary and two or more secondary nodes.
+
+   * - slave
+     - v
+     - subordinate
+     - You can subordinate one node to another.
```
[the remark this answers is not in the document]
Comment (atsansone) [reply source/style-guide/terminology/terms-for-global-audience.txt:351]: We don't use it, correct. I know that the phrase of "slaving" a device to another is known concept. I'm covering for future possibilities.

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 5]
```diff
@@ -305,6 +301,68 @@ If you create "named" users for extended examples or scenarios, use
...
+     - Examples
+
+   * - blacklist
+     - v
+     - remove, deny, block (access)
+     - MongoDB Atlas denies that IP address access to the database.
+
+   * - blacklist
+     - n
+     - access list, blocklist
```
[the remark this answers is not in the document]
Comment (atsansone) [reply source/style-guide/terminology/terms-for-global-audience.txt:334]: We actually use "access list". It's not my preference (I like block list or deny list), but it's [in the docs now](https://docs.atlas.mongodb.com/security/ip-access-list/index.html).

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 6]
```diff
@@ -305,6 +301,68 @@ If you create "named" users for extended examples or scenarios, use
 names that represent a variety of ethnic backgrounds, genders, and
 locations.
 
+.. _oppressive-language:
+
+Use Non-Oppressive Language
+---------------------------
+
+Language choice deals in connotation. To ignore that ignores the power
```
[the remark this answers is not in the document]
Comment (atsansone) [reply source/style-guide/terminology/terms-for-global-audience.txt:309]: Added the definition link. We're saying that words can have an implied meaning in addition to its stated meaning.

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 7]
```diff
@@ -169,83 +169,83 @@ industry and some possible alternatives.
      - Examples
 
    * - abort
-     - (v)
+     - v
```
[the remark this answers is not in the document]
Comment (atsansone) [reply source/style-guide/terminology/terms-for-global-audience.txt:172]: Just that it was what Rackspace did. I can replace all of that in a separate PR, if that's OK.

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 8]
```diff
@@ -305,6 +301,68 @@ If you create "named" users for extended examples or scenarios, use
...
+   :header-rows: 1
+
+   * - Term
+     - Part
+     - Alternative
+     - Examples
+
+   * - blacklist
+     - v
+     - remove, deny, block (access)
```
[the remark this answers is not in the document]
Comment (atsansone) [reply source/style-guide/terminology/terms-for-global-audience.txt:329]: We use "remove from access list" and "add to access list" in the docs right now. That's what was decided as acceptable terms. See this PR for details: https://github.com/10gen/cloud-docs/pull/1856

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 9]
```diff
@@ -305,6 +301,68 @@ If you create "named" users for extended examples or scenarios, use
...
+     - Replica sets have one primary and two or more secondary nodes.
+
+   * - slave
+     - v
+     - subordinate
+     - You can subordinate one node to another.
+
+   * - whitelist
+     - v
+     - add, allow, permit (access)
```
[the remark this answers is not in the document]
Comment (atsansone) [reply source/style-guide/terminology/terms-for-global-audience.txt:355]: Yeah, same as above. The accepted phrasing is "add to access list" and "remove from access list".

**Review by melissamahoney-mongodb — APPROVED** [review]

File: source/style-guide/terminology/terms-for-global-audience.txt (hunk 7 above)
[the remark this answers is not in the document]
Comment (jdestefano-mongo) [reply source/style-guide/terminology/terms-for-global-audience.txt:172]: Thanks for the context, I'm fine with that.

File: source/style-guide/terminology/terms-for-global-audience.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (jdestefano-mongo) [reply source/style-guide/terminology/terms-for-global-audience.txt:334]: I don't think your reply addressed either of my points above.

I do not understand how "access list" is an alternative for blacklist.

I asked if an alternative for blacklist is either "block list" or "blocklist". There is no space in the PR and a space in your comment.

File: source/style-guide/terminology/terms-for-global-audience.txt (hunk 9 above)
[the remark this answers is not in the document]
Comment (jdestefano-mongo) [reply source/style-guide/terminology/terms-for-global-audience.txt:355]: Can we add those phrases instead? That is much clearer to me.

File: source/style-guide/terminology/terms-for-global-audience.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (jdestefano-mongo) [reply source/style-guide/terminology/terms-for-global-audience.txt:351]: I would prefer to include advice on how we currently do things, like discourage using it as a verb.

File: source/style-guide/terminology/terms-for-global-audience.txt (hunk 8 above)
[the remark this answers is not in the document]
Comment (jdestefano-mongo) [reply source/style-guide/terminology/terms-for-global-audience.txt:329]: Can we add those phrases instead? That is much clearer to me.

**Review by jdestefano-mongo — COMMENTED** [review]
Left some comments, some things are still not clear to me

File: source/style-guide/terminology/terms-for-global-audience.txt (hunk 6 above)
[the remark this answers is not in the document]
Comment (jdestefano-mongo) [reply source/style-guide/terminology/terms-for-global-audience.txt]: I still can't wrap my head around the first two sentences and I don't think adding a definition is the right solution.

File: source/style-guide/terminology/terms-for-global-audience.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (atsansone) [reply source/style-guide/terminology/terms-for-global-audience.txt:334]: 1. The current documentation uses "access list" as a replacement for whitelist _and_ blacklist. It's just an access list now.
2. I grabbed "blocklist" from the [IETF RFC](https://tools.ietf.org/id/draft-knodel-terminology-00.html) on this topic to have something else besides _just_ access list. Again, I don't think "access list" is the best choice, but it is what we use for our documentation and user interface. 

image

[the remark this answers is not in the document]
Comment (jdestefano-mongo) [reply source/style-guide/terminology/terms-for-global-audience.txt]: Ok, thanks for the clarification. Worth adding a note that we currently use "access list" as a replacement for both?

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 10]
```diff
@@ -305,6 +301,87 @@ If you create "named" users for extended examples or scenarios, use
...
+   * - blacklist
+     - v
+     - remove (from an access list), deny (access to),
+       block (access to)
+     - MongoDB Atlas denies that IP address access to the database.
+
+   * - blacklist
+     - n
+     - access list, blocklist, exclude list
+     - That IP address has been removed from the access list to the
```
Comment (jdestefano-mongo) [inline source/style-guide/terminology/terms-for-global-audience.txt:342]:
```suggestion
     - That IP address has been removed from the access list for the
```

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 11]
```diff
@@ -305,6 +301,87 @@ If you create "named" users for extended examples or scenarios, use
...
+
+   * - whitelist
+     - v
+     - add (to an access list), allow (access to), permit (access to)
+     - MongoDB Atlas allows that IP address to access to the database.
+
+   * - whitelist
+     - n
+     - access list, allow list, allowed list
+     - That IP address has been added from the access list to the
```
Comment (jdestefano-mongo) [inline source/style-guide/terminology/terms-for-global-audience.txt:380]:
```suggestion
     - That IP address has been added from the access list for the
```

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 12]
```diff
@@ -305,6 +301,87 @@ If you create "named" users for extended examples or scenarios, use
 names that represent a variety of ethnic backgrounds, genders, and
 locations.
 
+.. _oppressive-language:
+
+Use Non-Oppressive Language
+---------------------------
+
+The words you choose can have meanings separate from its dictionary
```
Comment (jdestefano-mongo) [inline source/style-guide/terminology/terms-for-global-audience.txt:309]:
```suggestion
The words you choose can have meanings separate from their dictionary
```

File: source/style-guide/terminology/terms-for-global-audience.txt [hunk 13]
```diff
@@ -305,6 +301,87 @@ If you create "named" users for extended examples or scenarios, use
...
+definition. To ignore that ignores the power of speech and writing.
```
Comment (jdestefano-mongo) [inline source/style-guide/terminology/terms-for-global-audience.txt:310]:
```suggestion
definitions. To ignore that ignores the power of speech and writing.
```

**Review by jdestefano-mongo — COMMENTED** [review]
Basically LGTM, minor nits to take or leave. Left one discussion topic async.

**Review by jdestefano-mongo — APPROVED** [review]
lgtm, thanks @atsansone
